src/battlesnake.theserverproject.com/insert_clever_name/go.mod

```diff
@@ -0,0 +1,3 @@
+module battlesnake.theserverproject.com/insert_clever_name/snakeserver
```

Comment:
This path may be one directory too long? This way, the imports in snakeserver.go must be changed for the compilation to succeed.

Reply:
After more research, I still don't know what is correct here. Otherwise, I can still only get rid of errors by writing imports with relative paths. The full "battlesnake.theserverproject.com/insert_clever_name/snakeserver" import, for example, is not being interpreted in start_server.go, but "./snakeserver" is. Have to go to bed though.

Reply:
Okay, so I got the imports working with the absolute path from the go.mod file when I discovered changing priority from my GOPATH to the go.mod in my IDE. However, I still have to edit this line in the go.mod to be "battlesnake.theserverproject.com/insert_clever_name" so I still believe this is one directory too long.

Reply:
Changing to battlesnake.theserverproject.com/insert_clever_name *should* be the correct way of doing that, really what happened here was me fighting with everything for 5 hours just to get it running, so I pushed the first configuration that worked.

If you want to make that commit and merge feel free.